coins/bci.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Since this repo doesn't support BCI yet, can you remove this config so as to not cause confusion?